hibernate-core/src/main/java/org/hibernate/type/descriptor/java/ZonedDateTimeJavaDescriptor.java

```diff
@@ -59,7 +59,13 @@ public ZonedDateTime fromString(String string) {
 		}
 
 		if ( Timestamp.class.isAssignableFrom( type ) ) {
-			return (X) Timestamp.from( zonedDateTime.toInstant() );
+			/*
+			 * Workaround for HHH-13266 (JDK-8061577).
+			 * Ideally we'd want to use Timestamp.from( offsetDateTime.toInstant() ), but this won't always work.
```

Comment:
The line above has a copy-paste error. it should say ```Timestamp.from( zonedDateTime.toInstant() )```

Reply:
Right, will fix.